dnsaas/settings.py

```diff
@@ -227,3 +227,8 @@ def __getitem__(self, item):
         from settings_local import *  # noqa
     except ImportError:
         pass
+
+# list of domains for which RecordsRequests are instantly saved to Records
+# model
+# eg FORCE_AUTO_ACCEPT_DOMAINS = ['domain.pl', 'domain2.pl']
+FORCE_AUTO_ACCEPT_DOMAINS = []
```

Comment:
FORCE_AUTO_ACCEPT_DOMAINS = Unrestricted (field in the domain configuration)

Unrestricted -> Can users that are not owners of this domain add recordsto it without owner's permission?